Blog/GamingGuruBlog.Web/GamingGuruBlog.Domain/Services.cs

```diff
@@ -0,0 +1,34 @@
+﻿using GamingGuruBlog.Domain.Interfaces;
+using GamingGuruBlog.Domain.Models;
+
+namespace GamingGuruBlog.Domain
+{
+    public class Services
```

Comment:
Name this something else, like BlogPostService. Services should be separated by business function...in your case you might only have a couple (like UserService and BlogService and BlogPostService, or something like that) but don't have only one service in the domain